MSAL/src/native_auth/public/MSALNativeAuthUserAccountResult.swift

```diff
@@ -96,4 +97,47 @@ import Foundation
             }
         }
     }
+
+    /// Retrieves an access token for the account.
+    /// - Parameters:
+    ///   - client: The instance of Native Auth public client application.
+    ///   - forceRefresh: Ignore any existing access token in the cache and force MSAL to get a new access token from the service.
+    ///   - scopes: Optional. Permissions that should be included in the access token received after sign in flow has completed
+    ///   - correlationId: Optional. UUID to correlate this request with the server for debugging.
+    ///   - delegate: Delegate that receives callbacks for the Get Access Token flow.
+    public func getAccessToken(forceRefresh: Bool = false,
+                               scopes: [String]? = nil,
+                               correlationId: UUID? = nil,
+                               delegate: CredentialsDelegate) {
+
```

Comment:
Can you log here something like:
"Retrieving access token with scopes started". 
This can be useful for debugging purpose.
We usually log something at the beginning of the controller methods, but in this case, we don't call the controller